crates/compiler/load_internal/src/file.rs

```diff
@@ -884,9 +882,9 @@ enum PlatformPath<'a> {
 }
 
 #[derive(Debug)]
-struct PlatformData {
+struct PlatformData<'a> {
     module_id: ModuleId,
-    provides: Symbol,
+    provides: &'a [Loc<ExposedName<'a>>],
```

Comment:
This now gets resolved later in the pipeline, because the point where we used to look up the symbol no longer has a lock-free way to access `IdentIds` to turn the name into a `Symbol`. But since we don't use this symbol until the end when we're finishing specializations in single-threaded mode anyway, there's no need to lock to turn these into `Symbol`s at that stage!